### PR TITLE
Prune and update gh-actions workflows

### PR DESCRIPTION
**Pull request by maurolepore**
Relates to #393

This PR reduces two gh-action workflows, from being similar to
`usethis::use_github_action_check_full()` to being similar to
`usethis::use_github_action_standard()`. 

It not only reduced the number of checks (because it runs
in fewer platforms and versions of R) but also it updates
the workflows to more closely follow the workflows at 
r-lib/actions.

Notice that R 4.0.4 was released yesterday and gh-actions
may not reach this "too new" version of R. It is possible
that otherwise fine workflows now fail with R-release. If
the problem is indeed that R 4.0.4 is unavailable, then
we may just wait a few days in case the problem goes away.

The workflow that sources the web tool scripts is the 
skinniest, as the web tool runs only on one 
[platform](https://github.com/2DegreesInvesting/docker/blob/99e2844a31bcf97f6560245f32dff2b55cc3c746/system/Dockerfile#L1) and [version of R](https://github.com/2DegreesInvesting/docker/blob/99e2844a31bcf97f6560245f32dff2b55cc3c746/system/Dockerfile#L7).

* [Example run of source web tool scripts](https://github.com/2DegreesInvesting/PACTA_analysis/actions/runs/573053713)
* [Example run of R CMD check](https://github.com/2DegreesInvesting/PACTA_analysis/actions/runs/573053712)

### CODE REVIEW

**Comment by maurolepore** [comment]
The error I see on CI seems unrelated with our code. Look how similar it is with the error I get on my local ubuntu 20.04 running R 4.0.3 (not yet 4.0.4):

CI

```r
Error: Error: unable to load shared object '/home/runner/work/_temp/Library/stringi/libs/stringi.so':
  libicui18n.so.55: cannot open shared object file: No such file or directory
```

Local

```
> library(stringr)
Error: package or namespace load failed for ‘stringr’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/mauro/R/x86_64-pc-linux-gnu-library/4.0/stringi/libs/stringi.so':
  libicui18n.so.60: cannot open shared object file: No such file or directory
```

File: .github/workflows/source-web-tool-scripts.yaml [hunk 1]
```diff
@@ -11,20 +11,17 @@ on:
 name: Source web-tool scripts
 
 jobs:
-  Test-r-scripts:
+  R-CMD-check:
```
Comment (cjyetman) [inline .github/workflows/source-web-tool-scripts.yaml:14]: why is this changed to `R-CMD-CHECK` here in this action?

Comment (maurolepore) [reply .github/workflows/source-web-tool-scripts.yaml]: Thanks for noticing. I was sloppy when I copy-pasted from the R CMD check workflow. Fixed now.

**Review by cjyetman — APPROVED** [review]
cool, thanks! just for my information... do you have any idea what was crucial to getting the web tool scripts to run on the linux test?

**Comment by maurolepore** [comment]
No, haha. I'm not sure if anything that I did mattered. It may well be that coincidentally someone made the latest release of R (4.0.4) available for gh-actions on ubuntu.
